rfcs/protocols/asset-transfer/generic.md

```diff
@@ -23,38 +23,83 @@ This protocol uses as its building blocks:
+- `ReclaimAsset(<pledge-id>, <recipient-network-id>, <recipient-user-id>, <claim-status>)`: This first looks up the pledge corresponding to `<pledge-id>` from the ledger and validates it to ensure that it has expired. Then the claim status is parsed, which is an [AssetClaimStatus structure](../../formats/assets/transfer.md#representing-claims-on-pledged-assets) or a serialized form of it. (*Note*: this claim status is supposed to be sourced from the network where the given asset is to be transferred/claimed, and the associated proof is validated by the [interoperation module](../../models/infrastructure/interoperation-modules.md)). The claim status is then validated to ensure that the asset claimer matches the pledge recipient, that the pledger matches the original owner associated with the claim, that the claim tiem has expired, and that the claim was not made within the expiration time. The recipient must be identified both by a network ID (i.e., the network/ledger to which the asset is to be transferred) and a cryptographically verifiable identity (typically a certificate).
+- `GetAssetPledgeStatus(<pledge-id>, <recipient-network-id>, <recipient-user-id>)`: This returns the [AssetPledge structure](../../formats/assets/transfer.md#representing-an-asset-transfer-pledge) on the ledger (or a serialized form of it) corresponding to the `<pledge-id>`. It must validate the recipient network and user identities within the stored pledge against the passed parameter values before returning the information to the caller. It is meant to be used as a lookup function in a cross-network data sharing query (and hence will be accompanied by proof). The recipient's identity must be cryptographically verifiable (typically a certificate).
+- `GetAssetPledgeDetails(<pledge-id>)`: This returns the [AssetPledge structure](../../formats/assets/transfer.md#representing-an-asset-transfer-pledge) on the ledger (or a serialized form of it) corresponding to the `<pledge-id>`. It's purpose is similar to `GetAssetPledgeStatus` except that it doesn't perform validations. It is meant to be used as a lookup function within the network in which the asset has been pledged.
```

Comment:
This function is not core part of Asset Transfer protocol, and can be skipped from this section? As this section only explains the core functions for Asset Transfer?

Reply:
Moved to `fabric.md` with explanation.